Program.cs

```diff
@@ -60,64 +51,37 @@
 
    string one_to_nineteen (int num1) {

```

Comment:
Function name to start with capital letter. No underscore to be used for function and variable names.